prow/config/jobs/community.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     command: [make, test]
 
   - name: sync-org
-    image: gcr.io/k8s-prow/peribolos:v20200110-e950e4860
+    image: gcr.io/k8s-prow/peribolos:v20200114-29e07a334
```

Comment:
I think the update on this file is what's preventing me from just approving this PR?

Reply:
Makes sense, I think a reasonable workaround is just adding oncallers to the owners for this file? I am open to other options though